docs/content_management/field_types/field_type_reference/relationlistfield.md

```diff
@@ -107,3 +107,33 @@ $validators = [
     ]
 ];
 ```
+
+### Enable pagination in GraphQL
+
+To enable pagination for Relation List field type, set the `ibexa.graphql.schema.ibexa_object_relation_list.enable_pagination` parameter to `true`.
+
+!!! note
+
+    The pagination is enabled by default in [[= product_name =]] v5 and the parameter is removed.
+
+This allows you to query for only a subset of relations, as in the following example:
+
+``` json
```

Comment:
It's not JSON ;)
Is it how we create all graphql snippets?

Reply:
Thank you, there is a `graphql` variant as well (https://pygments.org/languages/) - will be fixed in a follow-up commit 🙇